src/billing/Project.js

```diff
@@ -558,33 +556,14 @@ const ProjectDetail = ({
             ]),
           ]),
     ]),
-    members: h(Fragment, [
-      isOwner &&
-        h(
-          NewUserCard,
-          {
-            onClick: () => setAddingUser(true),
-          },
-          [icon('plus-circle', { size: 14 }), div({ style: { marginLeft: '0.5rem' } }, ['Add User'])]
```

Comment:
`NewUserCard` renders an "Add User" button and ignores passed children, so this line got deleted.